src/Mod/Sketcher/App/planegcs/Geo.h

```diff
@@ -50,6 +48,9 @@ class Point
 };
 
 using VEC_P = std::vector<Point>;
+static const float pi = boost::math::constants::pi<float>();
+static const float pi_2 = boost::math::constants::pi<float>() / 2.0f;
+static const float pi_18 = boost::math::constants::pi<float>() / 18.0f;
```

Comment:
Can these be `constexpr`?

Reply:
The "Boost" mathematical constraints are already used: :https://github.com/FreeCAD/FreeCAD/blob/0bbb10864b11015e040860b549e93b1b708ab9b6/src/Gui/NaviCube.cpp#L633 .  
And ..in preparation for  #10650
M_PI  definition have a problem with library libe57 v3.1

Reply:
It's not a big deal, it just seems to me that if the underlying boost call is constexpr, we can make our constants a bit more flexible by also marking them as constexpr.

Reply:
Can you make these doubles, since they are used in double-precision arithmetic later on?